curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-002.md

```diff
@@ -0,0 +1,43 @@
+---
+id: 5f3313e74582ad9d063e3a38
+title: Part 2
+challengeType: 0
+isHidden: true
+---
+
+## Description
+<section id='description'>
+
+Add a `head` element within the `html` element, so you can add a `title` element. The title element's text should be `Camper Cafe Menu`.
```

Comment:
In the first project, we had them do this in two steps, but since we are trying to get into the CSS part as quickly as possible, I combined it.  I have no objections to splitting it, if you think it is necessary though.

Reply:
I'll leave it up to you. It's not necessary.

Reply:
OK, I will leave it for now.  We can always visit it again if we get the same feedback from others.